tracer/build/_build/Build.Steps.cs

```diff
@@ -349,7 +375,7 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
               var ddwafFileName = $"libddwaf.{ext}";
 
               var source = LibDdwafDirectory() / "runtimes" / sourceArch / "native" / ddwafFileName;
-              var dest = MonitoringHomeDirectory / sourceArch;
+              var dest = MonitoringHomeDirectory / "osx";
```

Comment:
Doesn't the waf need to support universal binaries for this to work? 🤔

Reply:
in this case, waf will be loaded on x64, but will fail to load in arm64. I told @robertpi to enable waf in arm64, then I will create the universal binary as well.